### PR TITLE
update cf-701C

### PR DESCRIPTION
**Pull request by freakin23**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [ ] I have tested my code.
- [ ] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [ ] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [ ] I have linked this PR to any issues that it closes.

### CODE REVIEW

File: solutions/silver/cf-701C.mdx [hunk 1]
```diff
@@ -118,10 +122,10 @@ public class Everywhere {
 ```
 
 </JavaSection>
-
 <PySection>
 
 ```py
+
```
Comment (SansPapyrus683) [inline solutions/silver/cf-701C.mdx]: ?

Comment (freakin23) [reply solutions/silver/cf-701C.mdx]: the bot fixed it.

**Comment by freakin23** [comment]
Can you check my other 2 prs as well

**Comment by SansPapyrus683** [comment]
hps is for ryan to merge
i'm still on the fence about the shortest paths one

**Comment by freakin23** [comment]
> hps is for ryan to merge i'm still on the fence about the shortest paths one

he approved the hps one and maybe left it for you to approve and merge.

as for the other one, I tried several other conditions but they all couldn't pass all TCs.

**Review by SansPapyrus683 — APPROVED** [review]

**Pull request merged**